### PR TITLE
Remove detail tab and update formatting

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
Signed-off-by: Andrew-Butler <Andrew.Butler@ibm.com>

@ckadner As mentioned in the meeting today
Screen Shot 2021-08-31 at 2 24 27 PM

### CODE REVIEW

**Comment by drewbutlerbb4** [comment]
@ckadner @Tomcli is this just flaky tests? Says the API failed to build, but nothing has been changed.

**Comment by ckadner** [comment]
> @ckadner @Tomcli is this just flaky tests? Says the API failed to build, but nothing has been changed.

This kind of error can happen when thirdparty libraries get update. Will take me a while to solve. Ignore it for the time being.

**Comment by ckadner** [comment]
In your screenshot, the "🏠 **Home**" tab is illuminated in the navigation, but the content shows a dataset?

File: dashboard/origin-mlx/src/styles/Models.css [hunk 1]
```diff
@@ -64,6 +64,10 @@ td>span.checked {
   color: #1ccdc7 !important;
 }
 
+.content {
+  padding-left: 10px;
```
Comment (ckadner) [inline dashboard/origin-mlx/src/styles/Models.css:68]: looks better with some space, but I think we can go bigger. How does 30px look? I think Github gives READMEs 32 px which looks nice.

**Review by ckadner — CHANGES_REQUESTED** [review]

**Comment by drewbutlerbb4** [comment]
@ckadner Here is 30px:
Screen Shot 2021-09-01 at 10 41 35 AM

**Review by ckadner — APPROVED** [review]
/lgtm

Thanks @drewbutlerbb4

**Comment by drewbutlerbb4** [comment]
@ckadner 
Well I tried to stop the merging process haha, I hadn't added the 30px in a commit yet. Just showed a picture. I can add it to a separate PR right now if you want it for the gifs?

**Comment by ckadner** [comment]
> @ckadner
> Well I tried to stop the merging process haha, I hadn't added the 30px in a commit yet. Just showed a picture. I can add it to a separate PR right now if you want it for the gifs?

Yes. Please :-)

**Comment by ckadner** [comment]
Follow-up PR to add the extra padding: #196

**Comment by ckadner** [comment]
> @ckadner @Tomcli is this just flaky tests? Says the API failed to build, but nothing has been changed.

#197

**Pull request merged**